src/backend/src/utils/utils.ts

```diff
@@ -24,3 +27,40 @@ export const convertStatus = (status: WBS_Element_Status): WbsElementStatus =>
 export const buildChangeDetail = (thingChanged: string, oldValue: string, newValue: string): string => {
   return `Changed ${thingChanged} from "${oldValue}" to "${newValue}"`;
 };
+
+const TOKEN_SECRET = process.env.TOKEN_SECRET || 'i<3security';
```

Comment:
I feel as though this should never have a default value. We'd want the service to crash if it isn't there. My general sentiment around default values is making sure that a default is truly a default, and not just a filler value. Why should 'i<3security' be our default token? And by embedding it directly in the code in a public repo that can be searched by bots, we're not doing ourselves any favors security wise.

For example, in java, we might use an Optional<Boolean>, so we can make it empty, true, or false vs. pretending false also represents non-existence.

Reply:
something about express_jwt errors if we don't have the token secret so this is just default for development. we are absolutely never using this in any production ever

Reply:
Solid reasoning. My point still stands. You could omit the variable in prod by accident and never notice because it would build properly. 

However, I see your point even if it sacrifices production integrity.

Reply:
totally agree, just don't want to deal with the headache for new devs